src/default.nix

```diff
@@ -258,7 +253,8 @@
         conditionalOverrides = packageOverrides;
       };
 
-    outputs = builder (builderArgs
+    build = dlib.warnIfIfd builder builder.build;
```

Comment:
Originally I had these warnings in `src/lib/builders.nix` and `src/lib/translators.nix` (`translate` and `build` was mapped to `dlib.warnIfIfd translator translate` and `dlib.warnIfIfd builder build` respectively), so I can change it back to that.

Reply:
I think it would be better to move it back to `src/lib/`, as otherwise if those were used in other places, the IFD warn logic would be missing by default.

Reply:
Done in 7554536400c8ef2747d87c9198d3849dba8cab43